perun-db/oracle.sql

```diff
@@ -368,6 +368,11 @@ create table auditer_consumers (
 create table services (
 	id integer not null,
 	name nvarchar2(128) not null,
+    description nvarchar2(1024),
+    delay integer not null default 10,
```

Comment:
For Oracle you must switch "NOT NULL" and "DEFAULT 10". Same for all Oracle entries. Please fix oracle changelog too.